--- a/src/pishutdown.c
+++ b/src/pishutdown.c
@@ -246,11 +246,10 @@
 
 static void button_handler (GtkWidget *widget, gpointer data)
 {
-    if (!strcmp (data, "shutdown")) system ("/usr/bin/pkill orca;/sbin/shutdown -h now");
-    if (!strcmp (data, "reboot")) system ("/usr/bin/pkill orca;/sbin/reboot");
+    if (!strcmp (data, "shutdown")) system ("gdbus call --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 --method org.freedesktop.login1.Manager.PowerOff true");
+    if (!strcmp (data, "reboot")) system ("gdbus call --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 --method org.freedesktop.login1.Manager.Reboot true");
     if (!strcmp (data, "exit"))
     {
-        system ("/usr/bin/pkill orca");
         if (!system ("pgrep labwc > /dev/null")) system ("/usr/bin/labwc -e");
         else system ("/usr/bin/pkill lxsession");
     }
