--- a/meson.build
+++ b/meson.build
@@ -2,7 +2,7 @@
     'wf-panel-pi',
     'c',
     'cpp',
-    version: run_command(['dpkg-parsechangelog', '--show-field', 'Version']).stdout().strip(),
+    version: run_command('head', '-n', '1', 'debian/changelog', check: true).stdout().split('(')[1].split(')')[0],
     meson_version: '>=0.60.0',
     default_options: [
         'cpp_std=c++17',
--- a/src/panel/plug_conf.c
+++ b/src/panel/plug_conf.c
@@ -227,7 +227,7 @@
     g_key_file_free (kf);
 
     // read system default
-    str = g_build_filename ("/etc/xdg/wf-panel-pi", wiz ? "wizard.ini" : "wf-panel-pi.ini", NULL);
+    str = g_build_filename ("/etc/xdg/rpd/wf-panel-pi", wiz ? "wizard.ini" : "wf-panel-pi.ini", NULL);
     kf = g_key_file_new ();
     g_key_file_load_from_file (kf, str, G_KEY_FILE_KEEP_COMMENTS | G_KEY_FILE_KEEP_TRANSLATIONS, NULL);
     g_free (str);
